src/services/WalletProvider/metamaskUtils/state.ts

```diff
@@ -1,4 +1,4 @@
-import { bool } from 'prop-types'
+import PropTypes from 'prop-types'
```

Comment:
The reason I don't like the desctruring here is that the names (`string`, `number`, `bool`, `object`) are similar or exactly the same as their TypeScript type counterparts. I know the compiler doesn't fall over this, because they're not _used_ as types here, but still it feels like bad practise to me.

Concerning the optimization, my understanding is that it doesn't matter here. It still includes the main 'prop-types' lib file which exports all of the other properties. The destructuring only allows us to access a subset after including them all.

It would actually make a difference if we could include the subset as follows:
```
import bool from 'prop-types/bool'
import string from 'prop-types/string'
```

But as far as I know, `prop-types` doesn't support that. Also for this lib it's probably such a micro optimization that it doesn't matter.